core/trino-main/src/main/java/io/trino/operator/project/PageProcessorMetrics.java

```diff
@@ -49,6 +60,10 @@ public Metrics getMetrics()
         if (hasFilter) {
             builder.put(FILTER_TIME, new DurationTiming(new Duration(filterTimeNanos, NANOSECONDS)));
         }
+        if (dynamicFilterInputPositions > 0) {
+            builder.put(DYNAMIC_FILTER_TIME, new DurationTiming(new Duration(dynamicFilterTimeNanos, NANOSECONDS)));
+            builder.put(DYNAMIC_FILTER_INPUT_POSITIONS, new LongCount(dynamicFilterInputPositions));
```

Comment:
No output positions?

Reply:
Output positions would be the same as the output positions of the Filter operator since this is now part of filter operator

Reply:
This will also be true for non-selective DF dropped in the middle?

Reply:
The flow of data is `Scan -> Static Filter -> Dynamic Filter`. Any dropping of non-selective dynamic filters happens internally in the dynamic filter and doesn't change the fact that output positions count of the dynamic filter is the same as the positions count of ScanFilterProject operator.